tweetlistener/index.py

```diff
@@ -45,13 +47,13 @@ def on_data(self, data):
                             with nostdout():
                                 minioClient.fput_object('colorization', filename_in, file_path_in)
 
-                            headers = {'X-Callback-Url': 'http://gateway:8080/async-function/tweetpic'}
+                            headers = {'X-Callback-Url': 'http://' + gateway_hostname + ':8080/async-function/tweetpic'}
```

Comment:
In the other PR I used the variable for the whole URL up to the slash, but I think this is OK, albeit inconsistent.